framework/db/pgsql/Schema.php

```diff
@@ -552,10 +552,13 @@ protected function findColumns($table)
             } elseif ($column->defaultValue) {
                 if (
                     in_array($column->type, [self::TYPE_TIMESTAMP, self::TYPE_DATE, self::TYPE_TIME], true) &&
-                    in_array(
-                        strtoupper($column->defaultValue),
-                        ['NOW()', 'CURRENT_TIMESTAMP', 'CURRENT_DATE', 'CURRENT_TIME'],
-                        true
+                    (
+                        in_array(
+                            strtoupper($column->defaultValue),
+                            ['NOW()', 'CURRENT_TIMESTAMP', 'CURRENT_DATE', 'CURRENT_TIME'],
+                            true
+                        ) ||
+                        preg_match('/TIMEZONE|NOW\(\)/i', $column->defaultValue)
```

Comment:
```suggestion
                        str_contains($column->defaultValue, '(')
```

Reply:
Thanks for you suggestion, but it will fail with text representation: `timezone(\'UTC\'::text, \'1970-01-01\')`

Reply:
Does it should be an `Expression`?
The value contains bracket `timezone(`

Reply:
1. I think yes, like `yiisoft/db` do
2. Oh, yes, my bad. For me `(` looks unobviously, I can't predict with other cases it may affect. `timezone` check is pretty straightforward.

Reply:
> I can't predict which other cases it may affect

Any other datetime function https://www.postgresql.org/docs/current/functions-datetime.html

For example, `current_timestamp(3)`